### PR TITLE
MNW-324 Remove Password Protection

### PR DESCRIPTION
**Pull request by loosefer**
## Issues

https://mnw.atlassian.net/browse/MNW-324

## Preview

image

### CODE REVIEW

File: packages/frontend/src/utils/encryption/index.ts [hunk 1]
```diff
@@ -21,3 +22,25 @@ export const encryptWallet = async (password: string): Promise<void> => {
...
+
+    const allAccounts = wallet.getAccountsLocalStorage();
+
+    for (let accountId in allAccounts) {
+        const keyPair = await encryptedkeyStore.getKey(CONFIG.NETWORK_ID, accountId) as KeyPairEd25519;
+        await encryptedkeyStore.removeKey(CONFIG.NETWORK_ID, accountId);
+        await decryptedkeyStore.setKey(CONFIG.NETWORK_ID, accountId, keyPair);
+    }
+
+    wallet.injectKeyStore(decryptedkeyStore);
```
Comment (NotEternal) [inline packages/frontend/src/utils/encryption/index.ts:45]: In the encryption flow we have a separate method `injectEncryptedKeyStore`. Won't it be more expected to move part of this code into `injectDecryptedKeyStore`?

File: packages/frontend/src/utils/encryption/index.ts [hunk 2]
```diff
@@ -21,3 +22,25 @@ export const encryptWallet = async (password: string): Promise<void> => {
...
+
+    const decryptedkeyStore = new nearApiJs.keyStores.BrowserLocalStorageKeyStore(
+        window.localStorage,
+        KEYSTORE_PREFIX
+    );
+
+    const allAccounts = wallet.getAccountsLocalStorage();
+
+    for (let accountId in allAccounts) {
+        const keyPair = await encryptedkeyStore.getKey(CONFIG.NETWORK_ID, accountId) as KeyPairEd25519;
```
Comment (NotEternal) [inline packages/frontend/src/utils/encryption/index.ts:40]: Some extra checks for key validation? For example some conditions with `ED25516_TAG` like in the encryption code.

File: packages/frontend/src/utils/encryption/EncryptedLocalStorage.ts [hunk 3]
```diff
@@ -9,22 +9,26 @@ export class EncrytedLocalStorage {
...
-    getItem(key) {
+    getItem(key: string): string {
         const encrypted = window.localStorage.getItem(key);
 
         return this.decrypt(encrypted);
     }
 
-    encrypt(value) {
+    removeItem(key: string): void {
+        window.localStorage.removeItem(key);;
```
[the remark this answers is not in the document]
Comment (loosefer) [reply packages/frontend/src/utils/encryption/EncryptedLocalStorage.ts:28]: fixed

File: packages/frontend/src/utils/encryption/index.ts (hunk 2 above)
Comment (loosefer) [reply packages/frontend/src/utils/encryption/index.ts:40]: Dont need, if key readable by keyStore, it is valid.

**Pull request merged**